src/styles/variables.css

```diff
@@ -0,0 +1,20 @@
+:root {
+  --blue-accent-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
+}
+
+@font-face {
+  font-family: "Roboto-Regular";
+  src: url("../fonts/Roboto-Regular.ttf") format("truetype");
+}
```

Comment:
if you are splitting your css - you may move `@font-face` to separate `typography.css` file 😉